packages/terra-time-input/src/TimeInput.jsx

```diff
@@ -571,6 +574,26 @@ class TimeInput extends React.Component {
     this.handleValueChange(event, TimeUtil.inputType.HOUR, this.state.hour.toString(), selectedKey);
   }
 
+  get a11yLabel() {
+    const { a11yLabel, intl } = this.props;
+    return a11yLabel || intl.formatMessage({
```

Comment:
Has a translations jira been logged for the new translation strings?

Reply:
Just now, yes.